gui/services/forms.py

```diff
@@ -1823,6 +1823,22 @@ def clean_iscsi_target_authgroup(self):
         return int(group)
 
 
+class TargetExtentDelete(Form):
+    '''
+    Delete form for Targets/Associated Targets
+    '''
+    def __init__(self, *args, **kwargs):
+        self.instance = kwargs.pop('instance', None)
+        super(TargetExtentDelete, self).__init__(*args, **kwargs)
+        if not self.data:
+            connected_targets = notifier().iscsi_connected_targets()
+            # Get target name from targetname/extentname combination
+            target_to_be_deleted = str(self.instance).split(' / ')[0]
```

Comment:
This make sense, but as the form is used for both target delete and target-extent delete, few more lines of code needs to be added, 
           ```
 if isinstance(self.instance, models.iSCSITarget):
     target_to_be_deleted = self.instance.iscsi_target_name
 elif isinstance(self.instance, models.iSCSITargetToExtent):
     target_to_be_deleted = self.instance.iscsi_target.iscsi_target_name
```
I have already commented what is being happening there. Do you still want me to make this change (as it add extra lines) or an elaborated comment should do?

This perfectly make sense in case of ExtentDelete.